### PR TITLE
chore: Update readme to remove self-hosted references

### PR DESCRIPTION
**Pull request by joethreepwood**
Removes self-hosted references, general updates

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,19 @@
...
-## PostHog is an open-source suite of product and data tools, built for engineers
+## PostHog is an all-in-one, open source platform for building better products
 
 - Specify events manually, or use autocapture to get started quickly
-- Analyze your data with visualizations and session recordings
-- Improve your product with A/B testing and feature flags
-- Keep control over your data by deploying PostHog on your infrastructure
-- Use apps to connect to external services and manage data flows
+- Analyze data with ready-made visualizations, or do it yourself with SQL
+- Gather insights by capturing session replays, console logs and network monitoring
```
Comment (ivanagas) [inline README.md:22]:
```suggestion
- Gather insights by capturing session replays, console logs, and network monitoring
```

File: README.md [hunk 2]
```diff
@@ -12,17 +12,19 @@
...
+- Improve your product with A/B testing that automatically analyzes performance
+- Safely roll out features to select users or cohorts with feature flags
+- Connect to external services and manage data flows with PostHog CDP
 
+PostHog can be deployed in the EU or US and is fully SOC 2 compliant. It's free to get started and your first 1 million events (and 15k replays) are free every month. We're constantly adding new features, such as user surveys!
```
Comment (ivanagas) [inline README.md:27]:
```suggestion
PostHog is available with hosting in the EU or US and is fully SOC 2 compliant. It's free to get started and your first 1 million events (and 15k replays) are free every month. We're constantly adding new features, such as user surveys!
```

File: README.md [hunk 3]
```diff
@@ -35,49 +37,49 @@
...
+### Open-source hobby deploy (Advanced)
 
+You can deploy a hobby instance in one line on Linux with Docker (recommended 4GB memory):
 
  ```bash 
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/deploy-hobby)" 
  ``` 
 
-Good for <100K events ingested monthly. See our [docs for more info and limitations](https://posthog.com/docs/self-host/open-source/deployment).
+Open source deployments should scale to approximately <100k events per month, after which we recommend migrating to a PostHog Cloud instance. See our [docs for more info and limitations](https://posthog.com/docs/self-host/open-source/deployment).
```
Comment (ivanagas) [inline README.md:52]:
```suggestion
Open source deployments should scale to approximately 100k events per month, after which we recommend migrating to a PostHog Cloud instance. See our [docs for more info and limitations](https://posthog.com/docs/self-host/open-source/deployment).
```

File: README.md [hunk 4]
```diff
@@ -35,49 +37,49 @@
...
 - **Plays nicely with data warehouses:** import events or user data from your warehouse by writing a simple transformation plugin, and export data with pre-built apps - such as [BigQuery](https://posthog.com/apps/bigquery-export), [Redshift](https://posthog.com/apps/redshift-export), [Snowflake](https://posthog.com/apps/snowflake-export), and [S3](https://posthog.com/apps/s3-expo)
 
 [Read a full list of PostHog features](https://posthog.com/product).
 
 ## Docs and support
 
-Read how to [deploy](https://posthog.com/docs/self-host), [integrate](https://posthog.com/docs/integrate), and [extend](https://posthog.com/docs/apps) PostHog in our [documentation](https://posthog.com/docs).
+Read how to [integrate](https://posthog.com/docs/integrate), and [extend](https://posthog.com/docs/apps) PostHog in our [documentation](https://posthog.com/docs).
```
Comment (ivanagas) [inline README.md:82]: I do think its useful to have hobby deploy docs here somewhere

**Review by ivanagas — APPROVED** [review]
Looks good

File: README.md (hunk 4 above)
Comment (joethreepwood) [reply README.md:82]:
```suggestion
Read how to [integrate](https://posthog.com/docs/integrate), and [extend](https://posthog.com/docs/apps) an open-source 'Hobby' deployment of PostHog in our [documentation](https://posthog.com/docs).
```

**Pull request merged**